test/features/section_report_multiple_groups/test_section_report_multiple_groups.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'feature_test'
+
+class TestSectionReport < FeatureTest
+  feature :section_report_multiple_groups do
+    params = {
+      type: :section,
+      layout_file: template_path,
+      params: {
+        start_page_number: 1,
+        groups: [
+          {
+            headers: {
+              header: {
+                items: {
+                  text: 'Header of Group 1'
+                }
+              }
+            },
+            details: 20.times.map {
+              {
+                id: :detail,
+                items: {
+                  text: 'Detail of Group 1'
+                }
+              }
+            }
+          },
+          {
+            headers: {
+              header: {
+                items: {
+                  text: 'Header of Group 2'
+                }
+              }
+            },
+            details: 10.times.map {
+              {
+                id: :detail,
+                items: {
+                  text: 'Detail of Group 2'
+                }
+              }
+            }
```

Comment:
Footer について何か変更があると良いですね。

```suggestion
            },
            footers: {
              footer: {
                display: false
              }
            }        
```

Reply:
https://github.com/standfirm/thinreports-generator/pull/46/commits/2c2580c56abc15065f8d9c55a02a7ae6982d9985 で修正しました